py3status/modules/process_status.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
 """
-Display if a process is running.
+Display status of a process on your system.
 
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 10)
-    format: default format for this module (default '{icon}')
-    full: if True, match against full command line (default False)
-    icon_off: show when process not running (default '■')
-    icon_on: show when process running (default '●')
-    process: process name to check for (default None)
+    format: display format for this module (default '{icon}')
+    full: match against the full command line (default False)
+    icon_off: show icon if process is not running (default '■')
```

Comment:
show __as__ icon ...

Reply:
That does not make sense (to me) as this is the only option to show (as). Anyhow, I pushed out something a bit different... and we can  `mergeformat` this soon or later to get rid of the descriptions / lines / nitpicks. Proceed with `mergeformat`? .__.